### PR TITLE
Clean up abandoned files in /tmp

### PR DESCRIPTION
**Pull request by ikaneshiro**
These changes are only within the context of unit tests with the
exception of a change to the SIF assembler to clean up a left over file
from building encrypted containers.

Signed-off-by: Ian Kaneshiro <iankane@umich.edu>

**Description of the Pull Request (PR):**

Write your description of the PR here. Be sure to include as much background,
and details necessary for the reviewers to understand exactly what this is
fixing or enhancing.

**This fixes or addresses the following GitHub issues:**

- Fixes #

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

**Review by mem — APPROVED** [review]
thanks

File: internal/pkg/build/files/copy_test.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func TestMakeParentDir(t *testing.T) {
 			defer os.RemoveAll(dir)
 
 			// concatenate test path with directory, do not use a join function so that we do not remove a trailing slash
-			path := dir + tt.path
+			path := dir + "/" + tt.path
```
Comment (gvallee) [inline internal/pkg/build/files/copy_test.go:73]: maybe use `filepath.Join()` here?

**Review by gvallee — CHANGES_REQUESTED** [review]
Only a minor comment

File: internal/pkg/build/files/copy_test.go (hunk 1 above)
Comment (ikaneshiro) [reply internal/pkg/build/files/copy_test.go:73]: `filepath.Join()` will remove trailing `/` from paths which are a significant component to paths for this function as it follows the convention of `cp` assuming that paths ending with a slash are directories.

Comment (gvallee) [reply internal/pkg/build/files/copy_test.go:73]: Sorry I missed the comment.

**Review by gvallee — APPROVED** [review]
lgtm

**Pull request merged**